src/server/service/batch.rs

```diff
@@ -334,7 +334,7 @@ impl<E: Engine, L: LockManager> ReqBatcher<E, L> {
                 BatchLimiter::new(
                     BatchableRequestKind::PointGet,
                     timeout,
-                    HistogramReader::new(GRPC_MSG_HISTOGRAM_VEC.kv_get.clone()),
+                    HistogramReader::new(GRPC_MSG_HISTOGRAM_VEC.with_label_values(&["kv_get"])),
```

Comment:
can we use static metrics for this?

Reply:
> can we use static metrics for this?

If we use static metrics here, there would be the only usage of a custom global static metric , seems a little waste.

Reply:
Don't worry, I think this is fine considering the performance (global histogram vec is the slowest)